src/smt/env.cpp

```diff
@@ -94,6 +94,14 @@ void Env::finishInit(smt::PfManager* pm)
   {
     d_ochecker.reset(new theory::quantifiers::OracleChecker(*this));
   }
+  if (d_options.base.statisticsAll)
+  {
+    d_statisticsRegistry->setStatsAll(true);
+  }
+  if (d_options.base.statisticsInternal)
+  {
+    d_statisticsRegistry->setStatsInternal(true);
+  }
```

Comment:
Might be better to just call it always in case the default changes for d_all and d_internal (from false to true) and needs to be disabled.
```suggestion
  d_statisticsRegistry->setStatsAll(d_options.base.statisticsAll);
  d_statisticsRegistry->setStatsInternal(d_options.base.statisticsInternal);

```